demo/remLoader.js

```diff
@@ -113,35 +113,30 @@ async function parseControlPoints(bytesArray, remShaderMaterial, FlatbufferModul
 
 
 async function createControlMeshes(controlPoints, remShaderMaterial, FlatbufferModule, animationEngine) {
-  let loadingBar = getLoadingBar();
-  let loadingBarTotal = getLoadingBarTotal(); 
+  const loadingBar = getLoadingBar();
+  const loadingBarTotal = getLoadingBarTotal(); 
 
-  let point;
-
-  let allSpheres = [];
-  let controlTimes = [];
+  const allSpheres = [];
+  const controlTimes = [];
   for(let ii=0, len=controlPoints.length; ii<len; ii++) {
     if (ii % 1000 == 0) {
       loadingBar.set(Math.max(ii/len * 100, loadingBar.value)); // update individual task progress
       await pause()
     }
-    point = controlPoints[ii];
-
-    var vertex = {x: point.pos().x(), y: point.pos().y(), z: point.pos().z()};
-    var radius = 0.25;//point.radius();
-    var timestamp = point.viz(new FlatbufferModule.Flatbuffer.Primitives.HideAndShowAnimation())
-       .timestamp(new FlatbufferModule.Flatbuffer.Primitives.ObjectTimestamp())
-       .value() - animationEngine.tstart;
-
-
-    var timestampArray = [];
-    for (let ii = 0; ii < 63; ii++) {
-      timestampArray.push(timestamp)
-    }
-
-    var sphereGeo = new THREE.SphereBufferGeometry(radius);
+    const point = controlPoints[ii];
+
+    const vertex = {x: point.pos().x(), y: point.pos().y(), z: point.pos().z()};
+    const radius = 0.25;//point.radius();
+    const timestamp = point.viz(new FlatbufferModule.Flatbuffer.Primitives.HideAndShowAnimation())
+    .timestamp(new FlatbufferModule.Flatbuffer.Primitives.ObjectTimestamp())
+    .value() - animationEngine.tstart;
+    
+    
+    const timestampArray = Array(64).fill(timestamp)
```

Comment:
It looks like the array is filled with numbers. If so, you could use `new Float64Array(64).fill(timestamp)`. That save space and is a little faster.

Reply:
Oh nice! I did not realize you could specify the type of array. I'll do that